tests/test_communication.py

```diff
@@ -272,7 +272,7 @@ def test_load_firmware_moco_exception(mocker, motion_controller):
 @pytest.mark.virtual
 def test_connect_servo_virtual():
     mc = MotionController()
-    mc.communication.connect_servo_virtual()
+    mc.communication.connect_servo_virtual(port=1062)
```

Comment:
Why do you need to change the port?

Reply:
Because the default port is already in use by a virtual drive created in the contftest.